grails-app/services/au/org/ala/bie/ExternalSiteService.groovy

```diff
@@ -182,6 +186,22 @@ class ExternalSiteService implements GrailsConfigurationAware {
         }
         return result
     }
+    /**
+     *
+     * @param content
+     * @param language
+     * @param title
+     * @return boolean
+     */
+    Boolean isContentInLanguage(String content, String language, String title){
+        LanguageDetector detector = new OptimaizeLangDetector().loadModels()
+        // remove any html tags to prevent mis-detection
+        def cleanedContent  = Jsoup.parse(content).text()
+        LanguageResult result = detector.detect(cleanedContent)
+        log.warn("Content Language Detection for " + title)
```

Comment:
This is going to generate two log messages for every request. Perhaps just warn if there is a mismatch?

Reply:
Updated.